fullstop/cli.py

```diff
@@ -282,8 +282,8 @@ def resolve_violations(config, comment, since, region, meta, remeta, limit, viol
 
     kwargs['accounts'] = kwargs.get('accounts') or config.get('accounts')
 
-    if not kwargs['accounts'] and not kwargs['type'] and not region:
-        raise click.UsageError('At least one of --accounts, --type or --region must be specified')
+    if all([not violation_ids, not kwargs['accounts'], not kwargs['type'], not region]):
+        raise click.UsageError('At least one of --accounts, --type, --region or --violation-ids must be specified')
```

Comment:
does it also fail if I did `fullstop configure` before and specified a set of accounts?